src/deb/tomcat7/bin/setenv.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+CATALINA_HOME=/usr/share/tomcat7
+
+# The first existing directory is used for JAVA_HOME (if JAVA_HOME is not
+# defined in $DEFAULT)
+JDK_DIRS="/usr/lib/jvm/java-6-openjdk /usr/lib/jvm/java-6-sun /usr/lib/jvm/java-1.5.0-sun /usr/lib/j2sdk1.5-sun /usr/lib/j2sdk1.5-ibm"
```

Comment:
I might actually delete this file entirely. it was generated with the private instance and I dont think it is in use.